template/workflows/manifests/.github/workflows/azure-kubernetes-service.yml

```diff
@@ -101,6 +100,13 @@ jobs:
           cluster-name: ${{ env.CLUSTER_NAME }}
           admin: 'false'
           use-kubelogin: 'true'
+        
+      # Checks if the AKS cluster is private
+      - name: Is private cluster
```

Comment:
Can you pull the private cluster changes out into a seperate PR first?

Reply:
on it